src/symbolize/gimli/parse_running_mmaps_unix.rs

```diff
@@ -8,7 +8,7 @@ use super::mystd::str::FromStr;
 use super::{OsString, String, Vec};
 
 #[derive(PartialEq, Eq, Debug)]
-pub(super) struct MapsEntry {
+pub struct MapsEntry {
```

Comment:
Do we need this to be able to be visible outside this crate? Using `pub(crate)` is preferable if `pub(super)` needs to be weakened.

Reply:
@workingjubilee : Should be fixed now. The original `pub(super)` seems to be working after some refactoring. Same for the other finding below.